cmd/puppeth/ssh.go

```diff
@@ -77,7 +77,23 @@ func dial(server string, pubkey []byte) (*sshClient, error) {
 		log.Warn("No SSH key, falling back to passwords", "path", path, "err", err)
 	} else {
 		key, err := ssh.ParsePrivateKey(buf)
-		if err != nil {
+		if string(err.Error()) == "ssh: cannot decode encrypted private keys" {
+
+			fmt.Printf("Enter the passphrase to decrypt your ssh key (no echo):\n>")
```

Comment:
`no echo` -> `won't be echoed` to keep it consistent with all other password read instances.

Reply:
Also Puppeth's "labels" generally take the form of questions, not statements, so I'd rather formulate this as:

`fmt.Printf("What's the decryption password for the SSH key? (won't be echoed)\n> ")`